src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  health = 100;
+  hidden = false;
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(beast) {
+    if (beast.hidden === false && beast instanceof Herbivore === true) {
```

Comment:
you can simplify it to:
```suggestion
    if (!beast.hidden && beast instanceof Herbivore) {
```